doc/index.rst

```diff
@@ -207,26 +215,70 @@ Pulling Docker image from an online hub
 
 You can pull any `Docker` image by specifying in `DOCKER_IMAGE` variable, as long as the following requirement is met:
 
-* `python-catkin-tools`, `python-pip`, `python-rosdep`, `python-wstool`
 * sources.list set up (`example <http://wiki.ros.org/kinetic/Installation/Ubuntu#Installation.2BAC8-Ubuntu.2BAC8-Sources.Setup_your_sources.list>`_).
+* `python-catkin-tools`, `python-pip`, `python-rosdep`, `python-wstool`. If your Docker image is missing any of the above libraries, then you can still pass their name by `ADDITIONAL_DEBS` (see `variables section <./index.rst#optional-environment-variables>`_).
+
+Some more notes:
+
+* Setting `DOCKER_IMAGE` is a bit tricky:
+   * disables the set-up of ROS based on `ROS_REPO` (or non-recommended `ROS_REPOSITORY_PATH`), and ROS_DISTRO.
+   * but `ROS_DISTRO` still needs to be set.  
+* Some common credentials such as `.docker`, `.ssh`, `.subversion` are passed from CI native platform to Docker container.
+
+Pass custom variables to Docker
+-------------------------------
+
+On CI platform usually some variables are available for the convenience. Since all checks using `industrial_ci` are NOT running directly on the operating system running on CI, but instead running on `Docker` where those variables are not defined, dozens of them are already passed for you (you can see `the list of those variables <https://github.com/ros-industrial/industrial_ci/blob/master/industrial_ci/src/docker.env>`_).
+
+Still, you may want to pass some other vars. `DOCKER_RUN_OPTS='-e MY_VARIABLE_VALUE'` should do the trick.
+You can even set it to a specific value: `DOCKER_RUN_OPTS='-e MY_VARIABLE_VALUE=42'` (format varies per CI platform. These are Gitlab CI example).
```

Comment:
This is the Travis format.
The Gitlab format (proper YAML!) would be:  `DOCKER_RUN_OPTS: '-e MY_VARIABLE_VALUE'`.

Reply:
In https://github.com/ros-industrial/industrial_ci/pull/234/commits/38db36ec2c5f77343837bc9224583e4cd95b8c32 I added a disclaimer in the new supported CI system section that the format in this doc is by default for Travis CI.